applications/FluidDynamicsApplication/python_scripts/vms_monolithic_solver.py

```diff
@@ -380,4 +380,4 @@ def CreateSolver(model_part, config):
         fluid_solver.linear_solver = linear_solver_factory.ConstructSolver(
             config.linear_solver_config)
 
-    return fluid_solver
+    return fluid_solver
```

Comment:
Please revert this file

Reply:
greetings dear @jcotela,

1. kindly note that today I downloaded kratos-master again, and then copied and pasted the vms_monolithic_solver file as is. I hope this will resolve this issue. If not please let me know and I will be pleasure to do any further modifications.

2. for the "documents/kratos_doxyfile", (I removed it by mistake when I run this file for reading it and then deleted it by mistake). Sorry about that, and I just added it today.

3. for the .bin file inside "tests/TwoFluidDarcyTest", I just removed it as well.

Thanks and best regards,